include/libaktualizr/config.h

```diff
@@ -28,6 +28,7 @@ struct PKCS11_slot_st;
 struct P11Config {
   boost::filesystem::path module;
   std::string pass;
+  std::string label;
```

Comment:
I think adding of a new param requires its serialization/deserialization so it can be actualy read from `sota.toml`, something like this https://github.com/foundriesio/aktualizr/blob/1daf2d5a6bc8b3fdef926922cf17bd0685f187bc/src/libaktualizr/config/base_config.cc#L48 and https://github.com/foundriesio/aktualizr/blob/1daf2d5a6bc8b3fdef926922cf17bd0685f187bc/src/libaktualizr/config/base_config.cc#L57

Reply:
strangely it just worked. I'll check base_config.cc

Reply:
```
cat z-local.toml | grep -A 4 p11
[p11]
label = "foobar"
[mike@t14s device]$ $AKLITE_EXE -c . staatus --loglevel 0 | grep p11 -A 8
[p11]
module = ""
pass = ""
uptane_key_id = ""
tls_ca_id = ""
tls_pkey_id = ""
tls_clientcert_id = ""

[tls]

```

Reply:
Afer I added `CopyFromConfig(label, "label", pt);` I get

```
[mike@t14s device]$ $AKLITE_EXE -c . staatus --loglevel 0 | grep p11 -A 8
[p11]
module = ""
pass = ""
label = "foobar"
uptane_key_id = ""
tls_ca_id = ""
tls_pkey_id = ""
tls_clientcert_id = ""
```